Sources/AloeStackView/AloeStackView.swift

```diff
@@ -236,6 +236,12 @@ open class AloeStackView: UIScrollView {
   /// This background color will be used for any new row that is added to the stack view.
   /// The default color is clear.
   open var rowBackgroundColor = UIColor.clear
+    
+  /// The highlight background color of rows in the stack view.
+  ///
+  /// This highlight background color will be used for any new row that is added to the stack view.
+  /// The default color is UIColor(red: 242 / 255, green: 242 / 255, blue: 242 / 255, alpha: 1).
+  open var hightlightRowBackgroundColor = UIColor(red: 242 / 255, green: 242 / 255, blue: 242 / 255, alpha: 1)
```

Comment:
It would be best if you could get the color of the default UITableviewCell provided by UIKit.
But I do not know how to get the system default colors.
If you have any idea how to do it, I would appreciate it.

Reply:
Yeah I don’t know of a good way either 😕 What you have here is fine though!